package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axelar-network/axelar-cgp-solidity",
-  "version": "3.1.1",
+  "version": "3.2.0",
```

Comment:
The version bump should be done separately. This may not be the very last PR for the release.

Reply:
Will remove that for now